packages/material-ui/src/ButtonBase/ButtonBase.test.js

```diff
@@ -687,9 +687,9 @@ describe('<ButtonBase />', () => {
             Hello
           </ButtonBase>,
         );
-
         const button = getByRole('button');
         button.focus();
+
         fireEvent.keyDown(document.activeElement || document.body, {
```

Comment:
This is a bug currently. Space should trigger click on keyUp not keyDown (following `<button />`.